vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -48,8 +48,8 @@ if ($windowsSku -eq "2019") {
             "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.6.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.4.0",
             "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.5.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.23.0",
```

Comment:
Has we upgraded it from 1.23 to 1.24 in production? If no, we should still keep the old version until v1.23.0 is not used in production.

Reply:
yes, 1.24 is current version in production now which is released one month ago.

Reply:
I think that we should cache the new image version for better perf next time and then update the new version in production.